share/eds/proofs_cache_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func TestCache(t *testing.T) {
 	ODSSize := 16
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*15)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
lets return 15 seconds back

Reply:
we are adding an additional test:

```
t.Run(fmt.Sprintf("RangeNamespaceData:%s", name), func(t *testing.T) {
				t.Parallel()
				testRangeNamespaceData(ctx, t, createAccessor, size)
			})
```
it does make sense that we will need a higher timeout?